config/mail.php

```diff
@@ -106,6 +106,21 @@
         'name'    => env('ADMIN_MAIL_NAME', 'Admin'),
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Global "Contact" Address
+    |--------------------------------------------------------------------------
+    |
+    | General contact address used in the footer of the email templates.
+    | Here, you may specify a name and address that is
+    | used globally for all e-mails that are sent by your application.
+    |
+    */
+    'contact' => [
+        'address' => env('CONTACT_MAIL_ADDRESS'),
+        'name'    => env('CONTACT_MAIL_NAME', 'Contact'),
```

Comment:
I think here it should be 

> "MAIL_FROM_ADDRESS"

 instead of 

> "CONTACT_MAIL_ADDRESS"

Reply:
Hi @amit-webkul 
If I put "MAIL_FROM_ADDRESS" then there will be no way to use a different address from the default sender, which most use "noreply@domain.com", as that key is used in the `from` configuration. The contact form could be used to give the customer an email address from the contact department. And if they want to use the same, then can put the same address in both fields.